jena-core/src/main/java/org/apache/jena/n3/N3JenaWriter.java

```diff
@@ -105,16 +80,20 @@ N3JenaWriterCommon chooseWriter()
     {
         // Choose the writer
         String writerName = JenaRuntime.getSystemProperty(propWriterName) ;
-        if ( writerName == null ||
-             writerName.equals("N3") || writerName.equals(n3WriterPrettyPrinter) )
+        if ( writerName == null )
+            return new N3JenaWriterPP() ;
+        if ( writerName.equals("N3") )
             return new N3JenaWriterPP() ;
         
-        if ( writerName.equalsIgnoreCase(n3WriterPlain) )
-            return new N3JenaWriterCommon() ;
-        
-        if ( writerName.equalsIgnoreCase(n3WriterTriples) ||
-             writerName.equalsIgnoreCase(n3WriterTriplesAlt) )
-            return new N3JenaWriterTriples() ;
+//        if ( writerName.equals(n3WriterPrettyPrinter) )
```

Comment:
I think the only difference now is that `writerName.equals(n3WriterPrettyPrinter)` used to work, and now it won't? But probably no harm I guess?